bridge/test/webf_test_env.cc

```diff
@@ -188,6 +188,10 @@ void TEST_flushUICommand(int32_t contextId) {
   clearUICommandItems(reinterpret_cast<void*>(page));
 }
 
+void TEST_CreateBindingObject(int32_t context_id, void* native_binding_object, int32_t type, void* args, int32_t argc) {
+
```

Comment:
remove unused test

Reply:
This function are simulate for dart methods.